### ISSUE TITLE
Preparation for 2020 version

### ISSUE THREAD

**Issue by sudo-give-me-coffee** [sudo-give-me-coffee/PhotoGIMP#7]
- [x] Create this issue
- [ ] Create a "Welcome screen"
- [ ] Create basic templates
- [ ] Create a new splash screen
- [ ] Replace current desktop integration with appimage-dialog

**Comment by Symbian9** [comment]
"PhotoGIMP 2020, `v1.0`" released:

- https://github.com/Diolinux/PhotoGIMP
  - https://github.com/Diolinux/PhotoGIMP/releases/tag/1.0

**Comment by sudo-give-me-coffee** [comment]
@Symbian9 As in the [ official post of the announcement](https://diolinux.com.br/2020/06/photogimp-2020.html) there is no mention of this project, and as I am over 1 year trying to make them at least publish something like "Hey, someone did an AppImage of our project, click here to download" and I didn’t get any recognition, and in that period they released and encouraged the snap version even though it’s outdated, it was clear that the Diolinux team doesn’t want to my work associated with them, so the PhotoGIMP AppImage in the next release will be completely detached from Diolinux's PhotoGIMP with another name, icon and splashscreen. Thanks for letting us know about the new version of PhotoGIMP by Diolinux

**Comment by Symbian9** [comment]
>  it was clear that the Diolinux team doesn’t want to my work associated with them, so the PhotoGIMP AppImage in the next release will be completely detached from Diolinux's PhotoGIMP with another name, icon and splashscreen.

Sad news. What would be then it renamed to? *PhotoIMP* (*PIMP* for short) maybe?

**Comment by sudo-give-me-coffee** [comment]
| Sad news. What would be then it renamed to?

The fact that led me to make this decision occurred 1 hour ago, I haven't thought about it yet. Anyway PhotoMP looks great, can I use it?

**Comment by Symbian9** [comment]
> Anyway PhotoMP looks great, can I use it?

Off course!

**Comment by sudo-give-me-coffee** [comment]
Yes, but first you need to update the Readme and close a forum thread

**Comment by Symbian9** [comment]
> you need to update the Readme and close a forum thread
>
> you

Not me, I'm not forum user.

**Comment by sudo-give-me-coffee** [comment]
Sorry, I made a typo, English is not my mother language
